hybrid/HybridSampleApps/AccountEditor/build.gradle.kts

```diff
@@ -7,7 +7,7 @@ plugins {
 
 dependencies {
     api(project(":libs:SalesforceHybrid"))
-    implementation("androidx.core:core-ktx:1.12.0")
+    implementation("androidx.core:core-ktx:1.9.0")
```

Comment:
Why are we going back to 1.9 ?

Reply:
Good eyes.  @brandonpage and I had chatted about this before the recent dependency updates.  There are some React Native community plugins that are still dependent on the Kotlin standard library as it was when paired with `core-ktx` 1.9.0.  Those classes have since moved to a new artifact when brought in by `1.12.0`.  So, that creates a duplicate class exception I don't believe we can avoid any other way.